Sprint9/Tema2.js

```diff
@@ -0,0 +1,9 @@
+// Scrieți o funcție care returnează un obiect nou fără proprietățile furnizate
+
+function creareObiectNou(obiect, proprietate) {
```

Comment:
```suggestion
function getObiectFaraProprietate(obiect, proprietate) {
```

(optional)
Doar citind functia ar fi bine sa intelegem ce face ea.
Numele "creareObiectNou" nu da de inteles ca al doilea parametru se refera la proprietatea care va fi stersa.